test/cpp/qps/driver.cc

```diff
@@ -241,8 +242,10 @@ std::unique_ptr<ScenarioResult> RunScenario(
     }
   }
 
-  // Setup the hosts and core counts
-  auto hosts_cores = get_hosts_and_cores(workers);
+  std::unordered_map<string, std::deque<int>> hosts_cores;
```

Comment:
Maybe this whole variable and its initialization need to be moved down below where it is actually used? That way you are just checking configure_core_lists once down below and then only declaring and setting up this variable if it's needed.